### ISSUE TITLE
About Gradio version

### ISSUE THREAD

**Issue by cjl0222** [ysy31415/unipaint#6]
Thanks for your great work! I'm wondering what is your Gradio version? When I'm running your `gradio_demo/demo.py`, and click the "initialize", "finetune" button, but that's not working. I tried Gradio 4.\*, but there's error. I think maybe your version is 3.\* ? Thanks a lot!

**Comment by ysy31415** [comment]
Hi, Thanks for your interest, I have checked my gradio version, which works on my server:
    - gradio==3.28.0
    - gradio-client==0.2.3
    
after clicking "initialize" button, there should be some info printed in terminal as follows:


`Global seed set to 42 `
`x shape:torch.Size([1, 3, 512, 512])`
`m shape:torch.Size([1, 1, 512, 512])`
`x_in shape:torch.Size([1, 3, 512, 512])`
`....`


after clicking "finetune" button, there should be a progress bar displayed in the terminal like follows:
`Fine-tune the model: 100%|██████████| 100/100 [00:42<00:00,  2.33it/s`


maybe you can adjust your package version and see whether the bug can be fixed.

**Comment by QuangHuyDN** [comment]
I have tried with those version of Gradio but still get the same problem. Any idea ? Thanks.

**Comment by chen29181** [comment]
> 我已经尝试过使用这些版本的 Gradio，但仍然遇到同样的问题。有什么想法吗？谢谢。

hh,把fastapi降级为0.99.0再试试
